#!/bin/bash
#$ -cwd
 
CHR=$1
CHUNK_START=`printf "%.0f" $2`
CHUNK_END=`printf "%.0f" $3`
RS_NUMBER=$4

# directories
ROOT_DIR=./
REF_DIR=/media/WD/common/imputation/data/
DATA_DIR=${ROOT_DIR}data_files/
RESULTS_DIR=${ROOT_DIR}results/
HAP_SAMP_DIR=${ROOT_DIR}sampled_haps/
 
# executable
IMPUTE2_EXEC=impute2
 
# parameters
NE=20000
 
## MODIFY THE FOLLOWING THREE LINES TO ACCOMODATE OTHER PANELS
# reference data files
GENMAP_FILE=${REF_DIR}ALL_1000G_phase1integrated_v3_impute/genetic_map_chr${CHR}_combined_b37.txt
HAPS_FILE=${REF_DIR}ALL_1000G_phase1integrated_v3_impute/ALL_1000G_phase1integrated_v3_chr${CHR}_impute.hap.gz
LEGEND_FILE=${REF_DIR}ALL_1000G_phase1integrated_v3_impute/ALL_1000G_phase1integrated_v3_chr${CHR}_impute.legend.gz
 
## THESE HAPLOTYPES WOULD BE GENERATED BY THE PREVIOUS SCRIPT
# best-guess haplotypes from phasing run
GWAS_BG_HAPS_FILE=${RESULTS_DIR}gwas_data-${RS_NUMBER}.phasing.impute2_haps
 
# main output file
OUTPUT_FILE=${RESULTS_DIR}gwas_data-${RS_NUMBER}.best_guess_haps_imputation.impute2
 
## impute genotypes from best-guess GWAS haplotypes
$IMPUTE2_EXEC \
   -m $GENMAP_FILE \
   -known_haps_g $GWAS_BG_HAPS_FILE \
   -h $HAPS_FILE \
   -l $LEGEND_FILE \
   -Ne $NE \
   -int $CHUNK_START $CHUNK_END \
   -o $OUTPUT_FILE \
   -allow_large_regions \
   -seed 367946
