
**[Writting Example](http://cmb.bnu.edu.cn/teachers/niudk_files/Good_wordings.htm)**

**[Metastasis-suppressor transcript destabilization through TARBP2 binding of mRNA hairpins](http://www.nature.com/nature/journal/v513/n7517/full/nature13466.html)**

Aberrant regulation of RNA stability has an important role in many disease states. Deregulated post-transcriptional modulation, such as that governed by microRNAs targeting linear sequence elements in messenger RNAs, has been implicated in the progression of many cancer types. A defining feature of RNA is its ability to fold into structures. However, the roles of structural mRNA elements in cancer progression remain unexplored. 

Here we performed an unbiased search for post-transcriptional modulators of mRNA stability in breast cancer by conducting whole-genome transcript stability measurements in poorly and highly metastatic isogenic human breast cancer lines. 

Using a computational framework that searches RNA sequence and structure space, we discovered a family of GC-rich structural cis-regulatory RNA elements, termed sRSEs for structural RNA stability elements, which are significantly overrepresented in transcripts displaying reduced stability in highly metastatic cells.

By integrating computational and biochemical approaches, we identified TARBP2, a double-stranded RNA-binding protein implicated in microRNA processing, as the trans factor that binds the sRSE family and similar structural elements—collectively termed TARBP2-binding structural elements (TBSEs)—in transcripts. TARBP2 is overexpressed in metastatic cells and metastatic human breast tumours and destabilizes transcripts containing TBSEs. Endogenous TARBP2 promotes metastatic cell invasion and colonization by destabilizing amyloid precursor protein (AP                                                                                                                    P)  and ZNF395 transcripts, two genes previously associated with Alzheimer’s and Huntington’s disease, respectively. We reveal these genes to be novel metastasis suppressor genes in breast cancer. 

The cleavage product of APP, extracellular amyloid-α peptide, directly suppresses invasion while ZNF395 transcriptionally represses a pro-metastatic gene expression program. The expression levels of TARBP2, APP and ZNF395 in human breast carcinomas support their experimentally uncovered roles in metastasis. Our findings establish a non-canonical and direct role for TARBP2 in mammalian gene expression regulation and reveal that regulated RNA destabilization through protein-mediated binding of mRNA structural elements can govern cancer progression.

**[β‐ but not γ‐secretase proteolysis of APP causes synaptic and memory deficits in a mouse model of dementia](http://embomolmed.embopress.org/content/4/3/171.abstract)**

A mutation in the BRI2/ITM2b gene causes loss of BRI2 protein leading to familial Danish dementia (FDD). BRI2 deficiency of FDD provokes an increase in amyloid‐β precursor protein (APP) processing since BRI2 is an inhibitor of APP proteolysis, and APP mediates the synaptic/memory deficits in FDD. APP processing is linked to Alzheimer disease (AD) pathogenesis, which is consistent with a common mechanism involving toxic APP metabolites in both dementias. We show that inhibition of APP cleavage by β‐secretase rescues synaptic/memory deficits in a mouse model of FDD. β‐cleavage of APP yields amino‐terminal‐soluble APPβ (sAPPβ) and β‐carboxyl‐terminal fragments (β‐CTF). Processing of β‐CTF by γ‐secretase releases amyloid‐β (Aβ), which is assumed to cause AD. However, inhibition of γ‐secretase did not ameliorate synaptic/memory deficits of FDD mice. These results suggest that sAPPβ and/or β‐CTF, rather than Aβ, are the toxic species causing dementia, and indicate that reducing β‐cleavage of APP is an appropriate therapeutic approach to treating human dementias. Our data and the failures of anti‐Aβ therapies in humans advise against targeting γ‐secretase cleavage of APP and/or Aβ.


Processing of the amyloid β protein precursor (AβPP) by the β and γ secretases leads to the production of two small peptides, amyloid β and the AβPP intracellular domain (AID, or called elsewhere AICD). Whereas the role of amyloid β in the pathogenesis of Alzheimer's disease has been studied extensively, only recently has information begun to accumulate as to the role of AID. Functions identified for AID include its ability to trigger apoptosis and a role in regulating gene transcription, particularly in combination with the AβPP binding protein Fe65. Here, we report that AID in combination with Janus kinase interacting protein-1 (JIP-1) can activate gene expression. We demonstrate that the mechanism is different from activation in combination with Fe65 by first showing that although Fe65 enters the nucleus in the absence of full-length AβPP, JIP-1 does not. Additionally, JIP-1-induced activation is Tip60 independent, whereas a complex with AID, Fe65, and Tip60 is formed for Fe65-induced activation. Finally, and probably most interestingly, we show that although the AβPP family members APLP1 and APLP2 (for amyloid β precursor-like protein) can cause activation in combination with Fe65, APLP1 and APLP2 show little or no activation in combination with JIP-1. This activity for the AID fragment may help explain the unique functions of AβPP relative to its other family members, and changes in gene expression found in Alzheimer's disease.

**[The Case for Junk DNA](http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1004351)**

With the advent of deep sequencing technologies and the ability to analyze whole genome sequences and transcriptomes, **there has been a growing interest in exploring putative functions of the very large fraction of the genome that is commonly referred to as “junk DNA.”** 

**Whereas this is an issue of considerable importance in genome biology, there is an unfortunate tendency for researchers and science writers to proclaim the demise of junk DNA on a regular basis without properly addressing some of the fundamental issues that first led to the rise of the concept.** In this review, we provide an overview of the major arguments that have been presented in support of the notion that a large portion of most eukaryotic genomes lacks an organism-level function. Some of these are based on observations or basic genetic principles that are decades old, whereas others stem from new knowledge regarding molecular processes such as transcription and gene regulation.

**[Synaptic dysregulation in a human iPS cell model of mental disorders](http://www.nature.com/nature/journal/vaop/ncurrent/full/nature13716.html)**

Dysregulated neurodevelopment with altered structural and functional connectivity is believed to underlie many neuropsychiatric disorders, and ‘a disease of synapses’ is the major hypothesis for the biological basis of schizophrenia.

Although this hypothesis has gained indirect support from human post-mortem brain analyses and genetic studies, little is known about the pathophysiology of synapses in patient neurons and how susceptibility genes for mental disorders could lead to synaptic deficits in humans. 

Genetics of most psychiatric disorders are extremely complex due to multiple susceptibility variants with low penetrance and variable phenotypes. Rare, multiply affected, large families in which a single genetic locus is probably responsible for conferring susceptibility have proven invaluable for the study of complex disorders. 

Here we generated induced pluripotent stem (iPS) cells from four members of a family in which a frameshift mutation of disrupted in schizophrenia 1 (DISC1) co-segregated with major psychiatric disorders and we further produced different isogenic iPS cell lines via gene editing. 

We showed that mutant DISC1 causes synaptic vesicle release deficits in iPS-cell-derived forebrain neurons. Mutant DISC1 depletes wild-type DISC1 protein and, furthermore, dysregulates expression of many genes related to synapses and psychiatric disorders in human forebrain neurons.

Our study reveals that a psychiatric disorder relevant mutation causes synapse deficits and transcriptional dysregulation in human neurons and our findings provide new insight into the molecular and synaptic etiopathology of psychiatric disorders.

**[A framework for the interpretation of de novo mutation in human disease](http://www.nature.com/ng/journal/vaop/ncurrent/full/ng.3050.html)**

Spontaneously arising (de novo) mutations have an important role in medical genetics. For diseases with extensive locus heterogeneity, such as autism spectrum disorders (ASDs), the signal from de novo mutations is distributed across many genes, making it difficult to distinguish disease-relevant mutations from background variation. Here we provide a statistical framework for the analysis of excesses in de novo mutation per gene and gene set by calibrating a model of de novo mutation. We applied this framework to de novo mutations collected from 1,078 ASD family trios, and, whereas we affirmed a significant role for loss-of-function mutations, we found no excess of de novo loss-of-function mutations in cases with IQ above 100, suggesting that the role of de novo mutations in ASDs might reside in fundamental neurodevelopmental processes. We also used our model to identify ~1,000 genes that are significantly lacking in functional coding variation in non-ASD samples and are enriched for de novo loss-of-function mutations identified in ASD cases.

**[Histone chaperone ASF1A is required for maintenance of pluripotency and cellular reprogramming](http://www.sciencemag.org/content/345/6198/822.full)**

Unfertilized oocytes have the intrinsic capacity to remodel sperm and the nuclei of somatic cells. The discoveries that cells can change their phenotype from differentiated to embryonic state using oocytes or specific transcription factors have been recognized as two major breakthroughs in the biomedical field. Here, we show that ASF1A, a histone-remodeling chaperone specifically enriched in the metaphase II human oocyte, is necessary for reprogramming of human adult dermal fibroblasts (hADFs) into undifferentiated induced pluripotent stem cell. We also show that overexpression of just ASF1A and OCT4 in hADFs exposed to the oocyte-specific paracrine growth factor GDF9 can reprogram hADFs into pluripotent cells. Our Report underscores the importance of studying the unfertilized MII oocyte as a means to understand the molecular pathways governing somatic cell reprogramming.


**[Developmental gene networks: a triathlon on the course to T cell identity](http://www.nature.com/nri/journal/v14/n8/full/nri3702.html)**

Cells acquire their ultimate identities by activating combinations of transcription factors that initiate and sustain expression of the appropriate cell type-specific genes. T cell development depends on the progression of progenitor cells through three major phases, each of which is associated with distinct transcription factor ensembles that control the recruitment of these cells to the thymus, their proliferation, lineage commitment and responsiveness to T cell receptor signals, all before the allocation of cells to particular effector programmes. All three phases are essential for proper T cell development, as are the mechanisms that determine the boundaries between each phase. Cells that fail to shut off one set of regulators before the next gene network phase is activated are predisposed to leukaemic transformation.


**[Altitude adaptation in Tibetans caused by introgression of Denisovan-like DNA](http://www.nature.com/nature/journal/v512/n7513/full/nature13408.html)**

As modern humans migrated out of Africa, they encountered many new environmental conditions, including greater temperature extremes, different pathogens and higher altitudes. These diverse environments are likely to have acted as agents of natural selection and to have led to local adaptations. One of the most celebrated examples in humans is the adaptation of Tibetans to the hypoxic environment of the high-altitude Tibetan plateau. A hypoxia pathway gene, EPAS1, was previously identified as having the most extreme signature of positive selection in Tibetans, and was shown to be associated with differences in haemoglobin concentration at high altitude. Re-sequencing the region around EPAS1 in 40 Tibetan and 40 Han individuals, we find that this gene has a highly unusual haplotype structure that can only be convincingly explained by introgression of DNA from Denisovan or Denisovan-related individuals into humans. Scanning a larger set of worldwide populations, we find that the selected haplotype is only found in Denisovans and in Tibetans, and at very low frequency among Han Chinese. **Furthermore, the length of the haplotype, and the fact that it is not found in any other populations, makes it unlikely that the haplotype sharing between Tibetans and Denisovans was caused by incomplete ancestral lineage sorting rather than introgression.** Our findings illustrate that admixture with other hominin species has provided genetic variation that helped humans to adapt to new environments.


**[Coordinated Cell Type–Specific Epigenetic Remodeling in Prefrontal Cortex Begins before Birth and Continues into Early Adulthood](http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1003433)**

Little is known about changes of post-mitotic neurons during this prolonged maturation period, including changes in epigenetic regulation, and more broadly, in genome organization and function. Such knowledge is critical for a deeper understanding of human development, cognitive abilities, and psychiatric diseases. Here, we identify 1,157 genomic loci in neuronal cells from the prefrontal cortex that show developmental changes in a chromatin mark, histone H3 trimethylated at lysine 4 (H3K4me3), which has been associated with regulation of gene expression. Interestingly, the overwhelming majority of these developmentally regulated H3K4me3 peaks were defined by rapid gain or loss of histone methylation during the late prenatal period and the first year after birth, followed by slower changes during early and later childhood and minimal changes thereafter.


**[ZMYND11 links histone H3.3K36me3 to transcription elongation and tumour suppression](http://www.nature.com/nature/journal/v508/n7495/full/nature13045.html#affil-auth)**

Recognition of modified histones by 'reader' proteins plays a critical role in the regulation of chromatin. H3K36 trimethylation (H3K36me3) is deposited onto the nucleosomes in the transcribed regions after RNA polymerase II elongation. In yeast, this mark in turn recruits epigenetic regulators to reset the chromatin to a relatively repressive state, thus suppressing cryptic transcription. **However, much less is known about the role of H3K36me3 in transcription regulation in mammals. This is further complicated by the transcription-coupled incorporation of the histone variant H3.3 in gene bodies.** Here we show that the candidate tumour suppressor ZMYND11 specifically recognizes H3K36me3 on H3.3 (H3.3K36me3) and regulates RNA polymerase II elongation. Structural studies show that in addition to the trimethyl-lysine binding by an aromatic cage within the PWWP domain, the H3.3-dependent recognition is mediated by the encapsulation of the H3.3-specific 'Ser 31' residue in a composite pocket formed by the tandem bromo-PWWP domains of ZMYND11. Chromatin immunoprecipitation followed by sequencing shows a genome-wide co-localization of ZMYND11 with H3K36me3 and H3.3 in gene bodies, and its occupancy requires the pre-deposition of H3.3K36me3. Although ZMYND11 is associated with highly expressed genes, it functions as an unconventional transcription co-repressor by modulating RNA polymerase II at the elongation stage. ZMYND11 is critical for the repression of a transcriptional program that is essential for tumour cell growth; low expression levels of ZMYND11 in breast cancer patients correlate with worse prognosis. Consistently, overexpression of ZMYND11 suppresses cancer cell growth in vitro and tumour formation in mice. **Together, this study identifies ZMYND11 as an H3.3-specific reader of H3K36me3 that links the histone-variant-mediated transcription elongation control to tumour suppression.**

**[Large-scale meta-analysis of genome-wide association data identifies six new risk loci for Parkinson's disease](http://www.nature.com/ng/journal/vaop/ncurrent/full/ng.3043.html)**

We conducted a meta-analysis of Parkinson's disease genome-wide association studies using a common set of 7,893,274 variants across 13,708 cases and 95,282 controls. Twenty-six loci were identified as having genome-wide significant association; these and 6 additional previously reported loci were then tested in an independent set of 5,353 cases and 5,551 controls. 

Of the 32 tested SNPs, 24 replicated, including 6 newly identified loci. Conditional analyses within loci showed that four loci, including GBA, GAK-DGKQ, SNCA and the HLA region, contain a secondary independent risk variant. 

In total, we identified and replicated 28 independent risk variants for Parkinson's disease across 24 loci. Although the effect of each individual locus was small, risk profile analysis showed substantial cumulative risk in a comparison of the highest and lowest quintiles of genetic risk (odds ratio (OR) = 3.31, 95% confidence interval (CI) = 2.55–4.30; P = 2 × 10−16). We also show six risk loci associated with proximal gene expression or DNA methylation.

**[C9orf72 repeat expansions cause neurodegeneration in Drosophila through arginine-rich proteins](http://www.sciencemag.org/content/early/2014/08/06/science.1256800.full)**

An expanded GGGGCC repeat in C9orf72 is the most common genetic cause of frontotemporal dementia and amyotrophic lateral sclerosis. A fundamental question is whether toxicity is driven by the repeat RNA itself and/or by dipeptide repeat proteins generated by repeat-associated, non-ATG translation. To address this question we developed in vitro and in vivo models to dissect repeat RNA and dipeptide repeat protein toxicity. Expression of pure repeats in Drosophila caused adult-onset neurodegeneration attributable to poly-(glycine-arginine) proteins. Thus, expanded repeats promoted neurodegeneration through neurotoxic proteins. Expression of individual dipeptide repeat proteins with a non-GGGGCC RNA sequence showed both poly-(glycine-arginine) and poly-(proline-arginine) proteins caused neurodegeneration. These findings are consistent with a dual toxicity mechanism, whereby both arginine-rich proteins and repeat RNA contribute to C9orf72-mediated neurodegeneration

------
The repeat expansion could cause disease by three possible mechanisms: i) toxic sense and/or antisense repeat RNA species that **sequester** key RNA-binding proteins, ii) toxic dipeptide repeat (DPR) proteins, generated by repeat-associated, non-ATG (RAN) translation, or iii) reduced expression of C9orf72. 

**The absence of a severe phenotype in a homozygous C9orf72 mutation case (7), and the lack of C9orf72 coding mutations (8) argue against loss-of-function as a primary mechanism.** Neuronal aggregates of RNA, termed RNA foci, generated from both sense and antisense repeat transcripts are frequent in C9FTD/ALS patient brain (9–13). The GGGGCC repeat can be translated in all sense and antisense frames, two of which encode the same DPR, resulting in five DPR proteins, all of which form inclusions in widespread brain regions (10, 12, 14–18). **It is therefore of fundamental importance to understand the contributions of repeat RNA and DPR proteins to C9orf72-mediated neurodegeneration.**


**[Chromatin state dynamics during blood formation](http://www.sciencemag.org/content/early/2014/08/06/science.1256271)**

Chromatin modifications are crucial for development, yet little is known about their dynamics during differentiation. Hematopoiesis provides a well-defined model to study chromatin state dynamics; however, technical limitations impede profiling of homogeneous differentiation intermediates.   

We developed a high sensitivity indexing-first chromatin immunoprecipitation approach (iChIP) to profile the dynamics of four chromatin modifications across 16 stages of hematopoietic differentiation. We identify 48,415 enhancer regions and characterize their dynamics.**We find that lineage commitment involves de novo establishment of 17,035 lineage-specific enhancers.** These enhancer repertoire expansions foreshadow transcriptional programs in differentiated cells. **Combining our enhancer catalog with gene expression profiles, we elucidate the transcription factor network controlling chromatin dynamics and lineage specification in hematopoiesis. Together, our results provide a comprehensive model of chromatin dynamics during development.**

**[Key narcolepsy–influenza vaccine findings retracted]()**

The work provided the first **substantiation** of an autoimmune mechanism for narcolepsy, which could explain the Pandemrix side effect, researchers say. The vaccine, used only in Europe, seems to have triggered the disease in roughly one out of 15,000 children who received it. The affected children carried a gene variant for a particular human leukocyte antigen (HLA) type—a molecule that presents foreign proteins to immune cells—considered necessary for developing narcolepsy.

In the 18 December 2013 issue of STM, **Mignot and colleagues reported that T cells from people with narcolepsy, but not from healthy controls, are primed to attack by hypocretin, a hormone that regulates wakefulness.** They also showed molecular similarities between fragments of the H1N1 virus and the hypocretin molecule and suggested that these fragments might fool the immune system into attacking hypocretin-producing cells.

**Up to 95% of the neurons that produce hypocretin are missing in people with narcolepsy, and many researchers believe that they're destroyed by a highly targeted autoimmune reaction, says Thomas Scammell, a neurologist at Harvard Medical School in Boston.** The Mignot paper was the first “seemingly nice, objective piece of evidence” to support that hypothesis.

The retraction states that Mignot and his colleagues were unable to replicate the results of the ELISpot assay, a widely used method for measuring how immune system cells such as T cells respond to fragments of foreign proteins, called antigens. Mignot told Science that while attempting to develop a diagnostic test for narcolepsy based on the assay, “my lab could not make the ELISpot test work.”

T cell assays in general are notoriously difficult to reproduce, says immunologist Outi Vaarala of the National Institute for Health and Welfare (THL) in Helsinki, who spent several weeks earlier this summer in Mignot's lab hoping to follow up on their finding. She's tried to replicate it herself in several ways, with no success.

**[Foxp2 Regulates Gene Networks Implicated in Neurite Outgrowth in the Developing Brain](http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1002145)**

Forkhead-box protein P2 is a transcription factor that has been associated with intriguing aspects of cognitive function in humans, non-human mammals, and song-learning birds.

Heterozygous mutations of the human FOXP2 gene cause a monogenic speech and language disorder.  
Reduced functional dosage of the mouse version (Foxp2) causes deficient cortico-striatal synaptic plasticity and impairs motor-skill learning.  
 
Moreover, the songbird orthologue appears critically important for vocal learning. 
Across diverse vertebrate species, this well-conserved transcription factor is highly expressed in the developing and adult central nervous system. 

*Very little is known about the mechanisms regulated by Foxp2 during brain development.*
**We used an integrated functional genomics strategy to robustly define Foxp2-dependent pathways, both direct and indirect targets, in the embryonic brain.**

Specifically, we performed genome-wide in vivo ChIP–chip screens for Foxp2-binding and thereby identified a set of 264 high-confidence neural targets under strict, empirically derived significance thresholds.  

The findings, coupled to expression profiling and in situ hybridization of brain tissue from wild-type and mutant mouse embryos, strongly highlighted gene networks linked to neurite development.  

We followed up our genomics data with functional experiments, showing that Foxp2 impacts on neurite outgrowth in primary neurons and in neuronal cell models. Our data indicate that Foxp2 modulates neuronal network formation, by directly and indirectly regulating mRNAs involved in the development and plasticity of neuronal connections.




**[Allelic specificity of Ube3a expression in the mouse brain during postnatal development.](http://onlinelibrary.wiley.com/doi/10.1002/cne.23507/full)**

Proteomic analysis of UBE3A-interacting proteins in heterologous systems **has revealed a limited number of isoform-specific UBE3A protein partners that could reflect (or perhaps even dictate) unique patterns of subcellular distribution (Martinez-Noel et al., 2012).** 

Whether such interactions occur in neurons, change over development, or affect UBE3A localization is not yet known.

This finding is seemingly at odds with our observation that neurons preferentially localize Ube3a to the nucleus as they mature. 


**[Enhancer loops appear stable during development and are associated with paused polymerase](http://www.nature.com/nature/journal/v512/n7512/full/nature13417.html)**

Developmental enhancers initiate transcription and are fundamental to our understanding of developmental networks, evolution and disease. 

**Despite their importance, the properties governing enhancer–promoter interactions and their dynamics during embryogenesis remain unclear.** 

At the β-globin locus, enhancer–promoter interactions appear dynamic and cell-type specific, whereas at the HoxD locus they are stable and ubiquitous, being present in tissues where the target genes are not expressed. 

The extent to which preformed enhancer–promoter conformations exist at other, more typical, loci and how transcription is eventually triggered is unclear. 

Here we generated a high-resolution map of enhancer three-dimensional contacts during Drosophila embryogenesis, covering two developmental stages and tissue contexts, at unprecedented resolution. Although local regulatory interactions are common, long-range interactions are highly prevalent within the compact Drosophila genome. 

Each enhancer contacts multiple enhancers, and promoters with similar expression, suggesting a role in their co-regulation. Notably, most interactions appear unchanged between tissue context and across development, arising before gene activation, and are frequently associated with paused RNA polymerase. 

**Our results indicate that the general topology governing enhancer contacts is conserved from flies to humans and suggest that transcription initiates from preformed enhancer–promoter loops through release of paused polymerase.**


**[Three-dimensional structure of human γ-secretase](http://www.nature.com/nature/journal/vaop/ncurrent/full/nature13567.html)**

The γ-secretase complex, **comprising** presenilin 1 (PS1), PEN-2, APH-1 and nicastrin, is a membrane-embedded protease that controls a number of important cellular functions through substrate cleavage. 

Aberrant cleavage of the amyloid precursor protein (APP) **results in** aggregation of amyloid-β, which accumulates in the brain and consequently causes Alzheimer’s disease. Here we report the three-dimensional structure of an intact human γ-secretase complex at 4.5 Å resolution, determined by cryo-electron-microscopy single-particle analysis. 

The γ-secretase complex comprises a horseshoe-shaped transmembrane domain, which contains 19 transmembrane segments (TMs), and a large extracellular domain (ECD) from nicastrin, which sits immediately above the hollow space formed by the TM horseshoe. 

Intriguingly, nicastrin ECD is structurally similar to a large family of peptidases exemplified by the glutamate carboxypeptidase PSMA. 

**This structure serves as an important basis for understanding the functional mechanisms of the γ-secretase complex.**


**[BRCA2 prevents R-loop accumulation and associates with TREX-2 mRNA export factor PCID2](http://www.nature.com/nature/journal/v511/n7509/full/nature13374.html)**  

Genome instability is central to ageing, cancer and other diseases. It is not only proteins involved in DNA replication or the DNA damage response (DDR) that are important for maintaining genome integrity: from yeast to higher eukaryotes, mutations in genes involved in pre-mRNA splicing and in the biogenesis and export of messenger ribonucleoprotein (mRNP) also induce DNA damage and genome instability. 

This instability is frequently mediated by R-loops formed by DNA–RNA hybrids and a displaced single-stranded DNA. **Here we show that the human TREX-2 complex, which is involved in mRNP biogenesis and export, prevents genome instability as determined by the accumulation of γ-H2AX (Ser-139 phosphorylated histone H2AX) and 53BP1 foci and single-cell electrophoresis in cells depleted of the TREX-2 subunits PCID2, GANP and DSS1.**

 We show that the BRCA2 repair factor, which binds to DSS1, also associates with PCID2 in the cell. The use of an enhanced green fluorescent protein-tagged hybrid-binding domain of RNase H1 and the S9.6 antibody did not detect R-loops in TREX-2-depleted cells, but did detect the accumulation of R-loops in BRCA2-depleted cells. 
 
**The results indicate that R-loops are frequently formed in cells and that BRCA2 is required for their processing. This link between BRCA2 and RNA-mediated genome instability indicates that R-loops may be a chief source of replication stress and cancer-associated instability.**



**[Deciphering the functions and regulation of brain-enriched A-to-I RNA editing](http://www.nature.com/neuro/journal/v16/n11/full/nn.3539.html)**

Adenosine-to-inosine (A-to-I) RNA editing, in which genomically encoded adenosine is changed to inosine in RNA, is catalyzed by adenosine deaminase acting on RNA (ADAR). 

**This fine-tuning mechanism is critical during normal development and diseases, particularly in relation to brain functions. A-to-I RNA editing has also been hypothesized to be a driving force in human brain evolution.** 

A large number of RNA editing sites have recently been identified, mostly as a result of the development of deep sequencing and bioinformatic analyses. Deciphering the functional consequences of RNA editing events is challenging, but emerging genome engineering approaches may expedite new discoveries. To understand how RNA editing is dynamically regulated, it is imperative to construct a spatiotemporal atlas at the species, tissue and cell levels. Future studies will need to identify the cis and trans regulatory factors that drive the selectivity and frequency of RNA editing. We anticipate that recent technological advancements will aid researchers in acquiring a much deeper understanding of the functions and regulation of RNA editing.

**[The DNA methylation landscape of human early embryos](http://www.nature.com/nature/journal/v511/n7511/full/nature13544.html#affil-auth)**

DNA methylation is a crucial element in the epigenetic regulation of mammalian embryonic development. However, its dynamic patterns have not been analysed at the genome scale in human pre-implantation embryos due to technical difficulties and the scarcity of required materials. 

Here we systematically profile the methylome of human early embryos from the zygotic stage through to post-implantation by reduced representation bisulphite sequencing and whole-genome bisulphite sequencing. 

**We show that the major wave of genome-wide demethylation is complete at the 2-cell stage, contrary to previous observations in mice. Moreover, the demethylation of the paternal genome is much faster than that of the maternal genome, and by the end of the zygotic stage the genome-wide methylation level in male pronuclei is already lower than that in female pronuclei.**

The inverse correlation between promoter methylation and gene expression gradually strengthens during early embryonic development, reaching its peak at the post-implantation stage. 

**Furthermore, we show that active genes, with the trimethylation of histone H3 at lysine 4 (H3K4me3) mark at the promoter regions in pluripotent human embryonic stem cells, are essentially devoid of DNA methylation in both mature gametes and throughout pre-implantation development.**

Finally, we also show that long interspersed nuclear elements or short interspersed nuclear elements that are evolutionarily young are demethylated to a milder extent compared to older elements in the same family and have higher abundance of transcripts, indicating that early embryos tend to retain higher residual methylation at the evolutionarily younger and more active transposable elements. Our work provides insights into the critical features of the methylome of human early embryos, as well as its functional relation to the regulation of gene expression and the repression of transposable elements.
